### ISSUE TITLE
postSubmitRedirect compares paths gotten through two different iron-router APIs

### ISSUE THREAD

**Issue by funkyeah** [meteor-useraccounts/iron-routing#13]
In the postSubmitRedirect [Router.current().route.path](https://github.com/meteor-useraccounts/iron-routing/blob/c99854a83d851f8cab947bbce2eb14aee316449c/lib/client.js#L181) is used. 

It's being compared against prevPath which is indrectly being set [here](https://github.com/meteor-useraccounts/iron-routing/blob/c99854a83d851f8cab947bbce2eb14aee316449c/lib/client.js#L50) using Router.current().url 

Very undependable results happen with these two API calls when the current route has a parameterized path. 

Often Router.current().url returns a full url and sometimes it returns a path.
Often Router.current().route.path() returns the path and sometimes is returns a `null`.

The latter two cases both cause the resetPwd/:token redirect route to break.

Why do these two return undependable results:

The [function signature](https://github.com/iron-meteor/iron-router/blob/devel/lib/route.js#L163) for path allows for passing `params` and `options`. If you call without params then these two lines of code in iron-router  [urls.js](https://github.com/iron-meteor/iron-url/blob/master/lib/url.js#L325) break when there is a named parameter.

Long story short, I will file a bug with iron-router but since that project is very inactive it might be best to use Iron.Location.get() instead of waiting for bugfix. Iron.Location.get()  seems to more consistently return the right path. At the very least the API method used to get the current path in the two spots in useraccounts/iron-routing should probably be the same.
